yorkie/src/androidTest/kotlin/dev/yorkie/core/TestUtils.kt

```diff
@@ -12,7 +12,7 @@ const val GENERAL_TIMEOUT = 3_000L
 fun createClient() = Client(
     "http://10.0.2.2:8080",
     unaryClient = OkHttpClient.Builder()
-        .protocols(listOf(Protocol.HTTP_1_1))
+        .protocols(listOf(Protocol.H2_PRIOR_KNOWLEDGE))
```

Comment:
we should stay on http 1 as http 2 is very unstable

Reply:
using http2 is outdated code. I just tested it temporarily.